pkgs/top-level/python-packages.nix

```
@@ -171,6 +171,8 @@ in {
 
   adguardhome= callPackage ../development/python-modules/adguardhome { };
 
+  advantage_air= callPackage ../development/python-modules/advantage_air { };
```

Please normalize the attribute name. According to the [python contributing guidelines](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines), package names within nix should follow [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names)
```suggestion
  advantage-air= callPackage ../development/python-modules/advantage-air { };
```

the pname can remain as-is though.